src/main/java/com/google/api/codegen/viewmodel/ApiMethodViewBuilder.java

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import java.util.List;
+
+// TODO(vchudnov-g): Consider making ApiMethodView an abstract class rather than an interface, and
+// including ApiMethodBuilder as an interface or abstract class within that.
```

Comment:
I don't think `ApiMethodView` needs to be an abstract class to contain this interface within it.

Reply:
Oh, right you are! Did s/`ApiMethodViewBuilder`/`ApiMethodView.Builder`/g